src/solvers/jesse_dg/elixir_esdg_compressible_Euler_1d.jl

```diff
@@ -0,0 +1,79 @@
+using OrdinaryDiffEq, Trixi
+
+using StructArrays
+using RecursiveArrayTools,ArrayInterface
+using LazyArrays
+
+using LinearAlgebra,BenchmarkTools,Plots,UnPack
+
+using Octavian,CheapThreads
+using StartUpDG
+
+include("trixi_interface.jl") 
+include("flux_differencing.jl")
+include("ModalESDG.jl")
+include("modal_esdg.jl")
+
+###############################################################################
+# semidiscretization 
+
+N = 3
+K1D = 32
+CFL = .1
+FinalTime = .7
+
+rd = RefElemData(Line(),N)
+VX,EToV = uniform_mesh(Line(),K1D)
+md = MeshData(VX,EToV,rd)
+md = make_periodic(md,rd)
+
+eqn = CompressibleEulerEquations1D(1.4)
+function LxF_dissipation(u_ll, u_rr, orientation, equations::CompressibleEulerEquations1D)
+    return .5 * max_abs_speed_naive(u_ll, u_rr, orientation, equations) * (u_ll-u_rr)
+end
```

Comment:
`LxF_dissipation` is the same as `DissipationLocalLaxFriedrichs()`: https://github.com/trixi-framework/Trixi.jl/blob/222fcd8cfed2ed71419a301e251a63bfe20efaea/src/equations/numerical_fluxes.jl#L139-L142

Reply:
Thanks for the heads up! Just changed it to use that instead.